### ISSUE TITLE
Mixed layer calc crashes google cloud

### ISSUE THREAD

**Issue by dhruvbalwada** [s-ragen/cmip6hack-so-project#8]
Just wanted to create this issue so we don't forget to finish the work. 
Currently the mixed layer calculation when using multiple models runs very inefficiently and makes the ocean.pangeo.io server crash. We need to fix this. Since ML calculation is a spatially local calculation, there is no reason for the crash to take place - maybe this can be solved by better chunking.

**Comment by s-ragen** [comment]
Good call! Thanks Dhruv. We should definitely revisit this...I'm going to have to focus on other research for a bit but I'm definitely keen to get back to this!
